patch/llpcFragColorExport.cpp

```diff
@@ -1376,13 +1374,15 @@ Value* FragColorExport::Run(
                     comps[i] = pUndefFloat;
                 }
 
-                std::vector<Attribute::AttrKind> attribs;
-                attribs.push_back(Attribute::ReadNone);
+                Attribute::AttrKind attribs[] = {
+                    Attribute::ReadNone
+                };
 
                 // Do packing
-                args.clear();
-                args.push_back(comps[0]);
-                args.push_back(comps[1]);
+                Value* args[] = {
+                    comps[0],
+                    comps[1]
+                };
                 comps[0] = EmitCall("llvm.amdgcn.cvt.pkrtz",
                                     m_pContext->Float16x2Ty(),
                                     args,
```

Comment:
The same as above. Place { comps[0], comps[1] } to EmitCall without defining args. This is more clear since the arg list is very simple.